examples/configs.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
```

Comment:
I think a better organisation would be:

```
oppen/
  default_config.rb
  …
wadler/
  default.rb
  eager_print.rb
```

I know that you'd like to see them all in the console printing the difference between them, but if you think about it closely:

1. We're deciding what's comparable for the user
2. We're forcing them to read a complex program to understand the example.

Decomposing the examples into separate folders allows us to showcase with the bare minimum (even if repetitive) to achieve a certain effect.

Moreover, they can serve for fast copy-pasta + experimentation …